spring-ai-docs/src/main/antora/modules/ROOT/pages/api/chat/azure-openai-chat.adoc

```diff
@@ -335,24 +335,23 @@ Next, create an `AzureOpenAiChatModel` instance and use it to generate text resp
 
 [source,java]
 ----
-var openAIClient = new OpenAIClientBuilder()
+var openAIClientBuilder = new OpenAIClientBuilder()
   .credential(new AzureKeyCredential(System.getenv("AZURE_OPENAI_API_KEY")))
-  .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"))
-  .buildClient();
+  .endpoint(System.getenv("AZURE_OPENAI_ENDPOINT"));
 
 var openAIChatOptions = AzureOpenAiChatOptions.builder()
   .deploymentName("gpt-4o")
   .temperature(0.4)
   .maxTokens(200)
   .build();
 
-var chatModel = new AzureOpenAiChatModel(this.openAIClient, this.openAIChatOptions);
+var chatModel = new AzureOpenAiChatModel(openAIClientBuilder, openAIChatOptions);
```

Comment:
Since this constructor method is deprecated and will be removed, I will update this to use the builder instead when merging.